specification/communication/data-plane/Common/stable/2022-07-13/common.json

```diff
@@ -84,6 +84,49 @@
         "modelAsString": true
       }
     },
+    "CallingCommunicationIdentifierModel": {
+      "description": "Calling communication identifier model",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/CommunicationIdentifierModel"
+        }
+      ],
+      "properties": {
+        "unknownIdentifiers": {
+          "$ref": "#/definitions/UnknownIdentifierModel",
+          "description": "The unknown identifier."
+        }
+      }
+    },
+    "UnknownIdentifierModel": {
```

Comment:
There is already an unknown identifier in ACS common, it's an identifier that has type unknown and only populates the rawId. Can we delete this?

Reply:
I removed it